bytecode-to-opcodes/opcodes.ts

```diff
@@ -0,0 +1,16 @@
+import { ethers } from 'hardhat';
+import { disassemble, Bytecode } from '@ethersproject/asm';
+
+export async function getBytecode(contract: string): Promise<Bytecode> {
+  const Factory = await ethers.getContractFactory(contract);
+  return disassemble(Factory.bytecode);
+}
+
+export function getOpcodes(bytecode: Bytecode): string {
```

Comment:
what this function actually does is "convert to opcodes and data string" ... maybe rename the function to reflect this?